tests/test_compat.py

```diff
@@ -277,3 +277,25 @@ def fun(eq):
     np.testing.assert_allclose(new_vals["B_max"], 2)
     np.testing.assert_allclose(new_vals["R0/a"], old_vals["R0/a"])
     np.testing.assert_allclose(new_vals["err"], old_vals["err"], atol=1e-10)
+
+
+@pytest.mark.unit
+@pytest.mark.solve
+def test_rotate_zeta():
```

Comment:
would also be good to add a test that evaluates R,Z at zeta=0 for the original eq and then at zeta=zeta0 for an eq rotated by zero0 and ensure they're the same

Reply:
also a test rotating by pi/nfp for a symmetric equilibria to make sure the result is also symmetric

Reply:
are the new ones qood or you prefer to evaluate R and Z positions?